Resources/Config.debug.plist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm considering the name of this file and how it works alongside our existing .xconfig files in `Source/Configs`. I think it will be confusing to have a a file named `Config.*.plist` in addition to things named Secrets, Environment, and a directory named Configs full of `.xcconfig` files. The `Config.plist` also only contains secrets, whereas our other configuration variables are not secret. 

I think ideally we would only have one configuration mechanism for secret and non-secret values, given that both xcconfig files and .plist files offer the same security guarantees. But that is probably out of scope for this PR. Maybe we could file a ticket to refactor down to one configuration mechanism for the future, and for now we could rename `Config.*.plist` to `Secrets.*.plist`?

Reply:
I filed #393 for this.